staging/src/k8s.io/dynamic-resource-allocation/go.sum

```diff
@@ -55,6 +55,7 @@ github.com/golang/glog v1.2.2/go.mod h1:6AhwSGph0fcJtXVM/PEHPqZlFeoLxhs7/t5UDAwm
 github.com/golang/protobuf v1.5.4 h1:i7eJL8qZTpSEXOPTxNKhASYpMn+8e5Q6AdndVa1dWek=
 github.com/golang/protobuf v1.5.4/go.mod h1:lnTiLA8Wa4RWRcIUkrtSVa5nRhsEGBg48fD6rSs7xps=
 github.com/google/btree v1.1.3/go.mod h1:qOPhT0dTNdNzV6Z/lhRX0YXUafgPLFUh+gZMl761Gm4=
+github.com/google/cadvisor v0.52.1/go.mod h1:OAhPcx1nOm5YwMh/JhpUOMKyv1YKLRtS9KgzWPndHmA=
```

Comment:
@pohly can you approve this for DRA side?

Reply:
the kubelet API module spreading cadvisor as a dependency is a bit suspect. *populating* the kubelet stats from cadvisor is totally fine, but embedding cadvisor types in the kubelet stats types means all kubelet API consumers now depend on cadvisor, which is… a lot

Reply:
Ooh great catch @liggitt

/hold

Reply:
We could copy it and transfer over